common/components/Forms/AddServiceForm/index.tsx

```diff
@@ -18,45 +18,11 @@ const AddServiceForm: React.FC<Props> = ({ form, edit, currentService }) => {
   const { MonthPicker } = DatePicker
   const initialValues = useInitialValues(currentService)
 
-  const date = Form.useWatch('date', form)
-  const domain = Form.useWatch('domain', form)
-  const street = Form.useWatch('street', form)
-
-  const { previousMonth } = usePreviousMonthService({
-    date,
-    domainId: domain,
-    streetId: street,
-  })
-
   useEffect(() => {
-    if (!previousMonth) {
-      form.setFieldsValue({
-        rentPrice: 0,
-        electricityPrice: 0,
-        waterPrice: 0,
-        waterPriceTotal: 0,
-        garbageCollectorPrice: 0,
-        inflicionPrice: 0,
-      })
-      return
-    }
-    const {
-      rentPrice,
-      electricityPrice,
-      waterPrice,
-      waterPriceTotal,
-      garbageCollectorPrice,
-      inflicionPrice,
-    } = previousMonth
     form.setFieldsValue({
-      rentPrice,
-      electricityPrice,
-      waterPrice,
-      waterPriceTotal,
-      garbageCollectorPrice,
-      inflicionPrice,
+      date: dayjs(),
     })
-  }, [previousMonth, form])
```

Comment:
Um... Where did all this logic go? As i understand it is used to setup initial values for new services based on previous one